src/python/cli_new/lib/cli/http.py

```diff
@@ -64,7 +73,7 @@ def get_json(addr, endpoint, condition=None, timeout=5, query=None):
         data = None
 
         try:
-            data = read_endpoint(addr, endpoint, query)
+            data = read_endpoint(addr, endpoint, config, query)
         except Exception as exception:
```

Comment:
Well, I understand that this code used to silently drop arbitrary errors since time immemorial, but probably now it is the right time to fix this:)

Reply:
Well, good that you speak it out. The loop also bother me. So I removed it and add a timeout into the read_endpoint function. :-) Hope it looks good now.